interface/modules/custom_modules/oe-module-prior-authorizations/public/reports/list_report.php

```diff
@@ -92,21 +121,52 @@ function openNewTopWindow(newpid) {
                         print "<td></td>";
                     }
                     print "<td>" . text($iter['auth_num']) . "</td>";
+                    print "<td>" . text($iter['cpt']) . "</td>";
                     print "<td>" . text($iter['start_date']) . "</td>";
                     print "<td>" . text($iter['end_date']) . "</td>";
-                    if (($iter['end_date'] < date('Y-m-d')) && ($iter['end_date'] !== '0000-00-00') && !empty($iter['auth_num'])) {
+
+                    if (($iter['end_date'] < date('Y-m-d')) && ($iter['end_date'] !== '0000-00-00') && (!empty($iter['auth_num']))) {
                         print "<td style='color: red'><strong>" . xlt('Expired') . "</strong></td>";
                         print "<td></td>";
+                        print "<td></td>";
                     } else {
-                        print "<td>" . text($iter['init_units']) . "</td>";
-                        $unitCount = $iter['init_units'] - $numbers;
+                        $initialUnits = (int)$iter['init_units'];
+                        print "<td>" . text($initialUnits) . "</td>";
+
+                        $usedUnits = $initialUnits - $numbers;
+                        $unitCount = $initialUnits - $numbers;
+                        if (($iter['end_date'] >= date('Y-m-d') || $iter['end_date'] === '0000-00-00') && !empty($iter['auth_num'])) {
+                            $total_initial_units += $initialUnits;
+                            $total_used_units += $usedUnits;
+                        }
+
                         if ($unitCount > 0) {
                             print "<td>" . text($unitCount) . "</td>";
                         } else {
                             print "<td>&nbsp</td>";
                         }
-                    }
 
+                        if ($initialUnits > 0) {
+                            $percentRemaining = round(($usedUnits / $initialUnits) * 100);
+                        } else {
+                            $percentRemaining = 0;
+                        }
+
+                        $barColor = '#4CAF50'; // Green: Full/Plenty remaining
+                        if ($percentRemaining <= 66) {
+                            $barColor = '#ffc107'; // Yellow: Getting low
+                        }
+                        if ($percentRemaining <= 33) {
+                            $barColor = '#dc3545'; // Red: Empty
+                        }
```

Comment:
```suggestion
                        $barColor = match(true) {
                            ($percentRemaining <= 33) => '#dc3545'; // Red: Empty
                            ($percentRemaining <= 66) => '#ffc107'; // Yellow: Getting low
                            default => '#4CAF50'; // Green: Full/Plenty remaining
                        };
```

You might also consider assigning the colors to variables or even an enum so the "empty/getting low/full" meaning can be gotten from the names.